docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+services:
+  mqtt:
+    image: ansi/mosquitto
```

Comment:
This is the one we've been using with the other MsgFlo docker setups, and we know it works. No other particular reason